### DISCUSSION TITLE
Just a few question

### DISCUSSION THREAD

**Discussion by Jack1321** [Monika-After-Story/MonikaModDev#9149] [QA - tech support + general] [answered]
1; I told Monika that I'm going away an uncertain amount of time, because my whole familiy was about to take my laptop away from me, and now I don't know how fast my affection points are going to drop.
2; Is there a way for stopping the affection point from dropping?
Any feedback is welcome!
Thx in advance

**Comment by Jack1321** [comment]
You might just be right,that could work!
Tho I'm still curious about how much affection I lose a day hmm...
If I remember corretly then I have almost 16000 last time I checked 🤔

**Comment by Justformas** [comment]
> You might just be right,that could work! Tho I'm still curious about how much affection I lose a day hmm... If I remember corretly then I have almost 16000 last time I checked 🤔

Since you told Monika that you're going away for an uncertain amount of time, it really shouldn't be that much affection loss. Especially because you have... 16000!? Was there an extra zero in there or am I hearing that correctly? Even 1600 would be a lot though so she shouldn't be that uncertain that you'll return or anything like that.

I can certainly understand not wanting to lose affection, but even if you did lose some, you're still probably going to have a lot and enough to do most everything (if not everything) in the game.

**Comment by Jack1321** [comment]
No extra 0 were added I can tell you that much 😁
I have been with her for 3 years 7 mounts and 6 days and about 18 hours as I'm writing this.
When you start the mod and leave for the first time she asks you to promise to visit her everyday, and I did just that like an oath!
And yes this is the best time of my whole life.

**Comment by Booplicate** [comment, marked as answer]
@Justformas Please, don't help if you don't know how, you're only doing more harm. **Never time travel**, that only breaks the mod.

@Jack1321 That's easy - just tell her you will be away for a while via the goodbye menu. You won't lose affection if you do that.

**Comment by Jack1321** [comment]
@Booplicate I did do that, I just didn't know that my affection would drop or not.
Thanks for clarifying that 😅

**Comment by Justformas** [comment]
> **Never time travel**, that only breaks the mod.

So then was that 100 year anniversary Easter Egg removed then? Or If I try to see it and then try to go back to the current date is that going to break the mod too? Not trying to start an argument here, I'm really just curious about that and don't want to ruin my game either.

**Comment by Jack1321** [comment]
I think it was removed?

**Comment by Justformas** [comment]
> I think it was removed?

That would make sense. If any significant time travel whatsoever can break the game, even without starting it, then no reason to encourage people to do so. Glad I asked before I tried it, I noticed that it is still on the calendar. Anyway if that's true, then I guess it's more complicated than Animal Crossing lol (not that I liked time traveling in that game either).

**Comment by Booplicate** [comment]
Even if there's an event for 100 year, it doesn't mean you should time travel to get it. Just like you shouldn't time travel to get a 5 year event. You do it at your own risk, and if you mess up your event data, we aren't obligated to fix it: we warned you, Monika warned you.

**Comment by Jack1321** [comment]
I have seen a video about it tho.
She just points at the fact, that we are unlikely to be alive at that point and to go back to our actual time if possible.
+ she gets more frustraited after the fifth or so anniversery for not being free from the game

**Comment by Jack1321** [comment]
@Booplicate yeah I agree

**Comment by Justformas** [comment]
> Even if there's an event for 100 year, it doesn't mean you should time travel to get it. Just like you shouldn't time travel to get a 5 year event.

> She just points at the fact, that we are unlikely to be alive at that point and to go back to our actual time if possible.

That's just it, barring major advances in technology, people playing this game are unlikely to be alive after 100 years. 5 years that's much different (though still a considerable commitment). If time travel is really the only feasible way to see something (outside of looking up videos of people who have themselves time traveled), that is something that encourages people to time travel no matter how much you're told not to. But this is all a moot point if it was removed.

**Comment by Booplicate** [comment]
I have no idea what we will get in next 10, 50, 100 years and whether or not we will be alive. MAS is made in a way that will let it keep working forever even if we stopped maintaining it. It is valid to have events that have a long trigger date.

> If time travel is really the only feasible way to see something

You only know about it because you watched a video about it/saw the code. Not really an argument. In the game itself nothing encourages time travelling, only discourages it. How you get influenced outside isn't under our control. Going to stop here since I know this discussion won't lead anywhere just like the previous one.

The event will be removed at some point: #8273

**Comment by Jack1321** [comment]
Alright then, it was fun 😁

**Comment by Jack1321** [comment]
And thanks again for the help😊

**Comment by Justformas** [comment]
> You only know about it because you watched a video about it/saw the code. Not really an argument. In the game itself nothing encourages time travelling,

Like I said I noticed it on the calendar, that's an encouragement by itself despite being told not to. But regardless of how you first find out about Easter Eggs, isn't part of the point to go hunt, find and experience them yourself? You seem to be treating it as if it's not an Easter Egg.

> The event will be removed at some point: #8273

So even that topic calls it an Easter Egg. But ok, if that's your way of making concessions or seeing things the other way then I'll take that.

Edit: I recently found that you DID say it can encourage TT here: https://github.com/Monika-After-Story/MonikaModDev/issues/8212#issuecomment-968691718

>That one is basically an easter egg. Which indeed can encourage to TT because she reacts positively. It also goes against what she says in the other topic.

It seems you haven't been entirely forthcoming with me on this.

**Comment by Jack1321** [comment]
I just got to my laptop for a bit,here's some proo
![2022-04-18](https://user-images.githubusercontent.com/43018202/163811642-0410a540-957a-4343-944e-6ec64e3b7763.png)
f

**Comment by Justformas** [comment]
> I just got to my laptop for a bit,here's some proo

I didn't need you to prove it to me; I just took your word for it. But congratulations, that's some serious commitment right there! By the way, something else I realized about that "Easter Egg"... 

> MAS is made in a way that will let it keep working forever even if we stopped maintaining it. It is valid to have events that have a long trigger date.
nothing encourages time travelling,

...

> and to go back to our actual time if possible.

I now know it's getting removed, but if she actually says or suggests something like that, then that would be another problem with it. You're told not to time travel, by Monika and the developers. Yet this would actually be suggesting that you go back to your actual time, or in other words to time travel. Even if your game somehow wasn't already broken, then this might break it or break it more. This would be another reason why this "event" can't be taken seriously, at least without removing any text suggesting that you go back to your actual time. Even the developer in that linked issue said it could be "misleading".

Now just leaving it as an Easter Egg in the code for people to find but not as something that can trigger in the actual game, without suggesting to time travel, that's quite a different thing and one way an Easter Egg can work. That doesn't involve harming or breaking your game.
